apispec/utils.py

```diff
@@ -98,4 +98,4 @@ def validate_swagger(spec):
                 stderr=subprocess.STDOUT,
             )
         except subprocess.CalledProcessError as error:
-            raise exceptions.SwaggerError(error.output.decode('utf-8'))
+            raise exceptions.PluginError(error.output.decode('utf-8'))
```

Comment:
Ah, good catch. I think the proper fix is to add `SwaggerError` to `exceptions`. A `PluginError` is raised for plugin-related code.